gaphor/ui/__init__.py

```diff
@@ -72,6 +72,12 @@ def app_activate(gtk_app):
         assert application
         if not application.sessions:
             application.get_service(launch_service).open()
+        else:
+            for session in application.sessions:
+                main_window = session.get_service("main_window")
+                if main_window.window:
+                    main_window.window.present()
+                    break
```

Comment:
This is a really nice idea. However, instead of showing a random session, we can also show the last window that has been active.

```suggestion
        elif gtk_app.get_active_window():
            gtk_app.get_active_window().present()
```

Reply:
Yes, I was thinking about that. I updated the code.